people.md

```diff
@@ -7,18 +7,13 @@ Names are followed by GitHub usernames.
 - Rob Woodcock, @woodcockr ([CSIRO](https://www.csiro.au/))
 - Paul Haesler, @SpacemanPaul ([CSIRO](https://www.csiro.au/))
 - Kirill Kouzoubov, @Kirill888 (independent)
-- Alex Leith, @alexgleith ([Geoscience Australia](http://www.ga.gov.au/))
```

Comment:
You don't have to be affiliated with a supporting organisation to be a member of the steering council.
    You don't have to attend meetings to be a member of the SC (although obviously it's encouraged).
    I know for a fact @alexgleith is still actively contributing to SC conversations on Slack, so for all intents and purposes he is still a member of the SC.